front-end/src/Scenes/LogIn/Containers/logIn.css

```diff
@@ -0,0 +1,27 @@
+.layout{
```

Comment:
It will be perfect here do something like 
```
.login-page .layout {
...
}
.login-page .header {
...
}
```

or as you have SCSS preprocessor (at least I heard about it)
you can do even easier:
```
.login-page {
  .layout {
   ...
  }
  .header {
   ...
  }
  ...
}
```

Reply:
So one more time, why I it is important, why I annoying with that:
you already working in as with a team and the project isn't small
So you will have header (as an example) on all pages across application,
somewhere you want it to be black and on another pages it must be gray
so you will got a conflict,
or another situation: for some specific page your teammate made header transparent, but you will merge the code and by mistake all becomes in "hybrid" state. So that isn't what we want, isn't it?
I'm ok with situation that just header can have same style across the pages, but in that case we need to declare it not in login.css but in global.css or global-header.css 

And yea I know about css-modules, but don't get used to the good, because mostly in the real world project are bad.
And by the way css-modules is not a magic bullet at all ;)

Reply:
--- 
That is not change request if so and at least for current PR, just for you information.